server/src/core/event/event.resolver.ts

```diff
@@ -0,0 +1,49 @@
+import { Resolver, Mutation, Args, Context } from '@nestjs/graphql';
+import { EventService } from './event.service';
+import { Event } from './event.entity';
+import { CreateEventInput } from './dto/create-event.input';
+import axios from 'axios';
+import { OptionalJwtAuthGuard } from 'src/guards/optional-jwt.auth.guard';
+import { UseGuards } from '@nestjs/common';
+import { AuthWorkspace } from 'src/decorators/auth-workspace.decorator';
+import { User, Workspace } from '@prisma/client';
+import { AuthUser } from 'src/decorators/auth-user.decorator';
+import { anonymize } from 'src/utils/anonymize';
+
+@UseGuards(OptionalJwtAuthGuard)
```

Comment:
@FelixMalfait To use `AuthWorkspace` and `AuthUser` decorators, `JwtAuthGuard` is mandatory